docker-compose.yml

```diff
@@ -0,0 +1,139 @@
+version: "3.8"
+
+services:
+
+  postgres_druid:
+    container_name: postgres_druid
+    image: postgres:13
+    volumes:
+      - metadata_data:/var/lib/postgresql/data   
+    environment:
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD_DRUID}
+      - POSTGRES_USER=${POSTGRES_USER_DRUID}
+      - POSTGRES_DB=${POSTGRES_DB_DRUID}
+
+  postgres_harmony:
+    container_name: postgres_harmony
+    image: postgres:13
+    ports:
+      - 51432:5432
+    volumes:
+      - ./database:/var/lib/postgresql/data   
+    environment:
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD_HARMONY}
+      - POSTGRES_USER=${POSTGRES_USER_HARMONY}
+      - POSTGRES_DB=${POSTGRES_DB_HARMONY}
+
+  # Need 3.5 or later for container nodes
+  zookeeper:
+    container_name: zookeeper
+    image: zookeeper:3.5
+    environment:
+      - ZOO_MY_ID=1
+
+  coordinator:
+    image: apache/druid:0.20.0
+    container_name: coordinator
+    volumes:
+      - ./storage:/opt/data
+      - coordinator_var:/opt/druid/var
+    depends_on: 
+      - zookeeper
+      - postgres_druid
+    ports:
+      - "8081:8081"
+    command:
+      - coordinator
+    env_file:
+      - environment
+
+  broker:
+    image: apache/druid:0.20.0
+    container_name: broker
+    volumes:
+      - broker_var:/opt/druid/var
+    depends_on: 
+      - zookeeper
+      - postgres_druid
+      - coordinator
+    ports:
+      - "8082:8082"
+    command:
+      - broker
+    env_file:
+      - environment
+
+  historical:
+    image: apache/druid:0.20.0
+    container_name: historical
+    volumes:
+      - ./storage:/opt/data
+      - historical_var:/opt/druid/var
+    depends_on: 
+      - zookeeper
+      - coordinator
+      - postgres_druid
+    ports:
+      - "8083:8083"
+    command:
+      - historical
+    env_file:
+      - environment
+
+  middlemanager:
+    image: apache/druid:0.20.0
+    container_name: middlemanager
+    volumes:
+      - ./storage:/opt/data
+      - middle_var:/opt/druid/var
+    depends_on: 
+      - zookeeper
+      - postgres_druid
+      - coordinator
+    ports:
+      - "8091:8091"
+    command:
+      - middleManager
+    env_file:
+      - environment
+
+  router:
+    image: apache/druid:0.20.0
+    container_name: router
+    volumes:
+      - router_var:/opt/druid/var
+    depends_on:
+      - zookeeper
+      - postgres_druid
+      - coordinator
+    ports:
+      - "8888:8888"
+    command:
+      - router
+    env_file:
+      - environment
+
+  harmony_app:
+    build: .
+    container_name: harmony_app
+    ports:
+      - "5000:5000"
+    volumes:
+      - .:/zenysis
+    depends_on:
+      - middlemanager
+      - zookeeper
+      - coordinator
+      - broker
+      - historical
+      - postgres_druid
+      - postgres_harmony
```

Comment:
Might be useful to add a comment that these dependencies are only needed when you are running the druid cluster *locally*. For a production setup, it is expected that Druid will be hosted independently of the Harmony web server for durability and performance reasons.